MediaBrowser.Providers/Manager/MetadataService.cs

```diff
@@ -672,6 +672,8 @@ public bool CanRefreshPrimary(Type type)
             }
 
             var hasLocalMetadata = false;
+            var replaceImages = Enum.GetValues<ImageType>().ToList();
```

Comment:
I believe the enum values should be cached as it uses reflection to populate the array

Reply:
Implemented cache in latest commit